### PR TITLE
control panel UI tweaks

### PR DESCRIPTION
**Pull request by mattearnshaw**
- [ ] Add coverage for whatever new functionality, to a JUnit test if it's backend, to a Casper Test if it's frontend
- [ ] Make sure both push and pr travis builds are passing before asking for a review

hopefully these style changes (based on user feedback and https://github.com/openworm/org.geppetto.frontend/issues/555) are fairly uncontroversial

* tooltips on parameter boxes if project not persisted or experiment is complete
* style parameter inputs for experiments in design mode so that they are obviously inputs and vice versa
* sign-in icon for parameters view replaced by sliders icon

@

### CODE REVIEW

File: src/main/webapp/js/components/interface/controlPanel/controlpanel.js [hunk 1]
```diff
@@ -152,11 +152,23 @@ define(function (require) {
...
+                content: content
+            });
+        },
+
         componentDidMount: function () {
             // listen to experiment status change and trigger a re-render to refresh input / read-only status
             GEPPETTO.on(GEPPETTO.Events.Experiment_completed, this.refresh, this);
             GEPPETTO.on(GEPPETTO.Events.Experiment_running, this.refresh, this);
             GEPPETTO.on(GEPPETTO.Events.Experiment_failed, this.refresh, this);
+            if (!GEPPETTO.UserController.hasPersistence())
```
Comment (tarelli) [inline src/main/webapp/js/components/interface/controlPanel/controlpanel.js]: @mattearnshaw should this check first that the deployment is using a persistence bundle?

Comment (mattearnshaw) [reply src/main/webapp/js/components/interface/controlPanel/controlpanel.js]: @tarelli good point. `GEPPETTO.UserController.hasWritePermissions` ?

Comment (tarelli) [reply src/main/webapp/js/components/interface/controlPanel/controlpanel.js]: @mattearnshaw mmm I think that method you are already using does what I'm saying but not what you thought it was doing... I think it should be
`if (GEPPETTO.UserController.hasPersistence() && !window.Project.persisted)`
but I haven't tested it.

Comment (mattearnshaw) [reply src/main/webapp/js/components/interface/controlPanel/controlpanel.js]: @tarelli thanks, fixed

**Comment by tarelli** [comment]
@mattearnshaw can we update also the docs? http://docs.geppetto.org/en/latest/settingParameters.html
Feel free also to improve it, it's a bit thin :)

**Pull request merged**